features/financial_assistance/step_definitions/other_questions_page_steps.rb

```diff
@@ -104,6 +104,15 @@
   expect(page).to have_content('Has this person applied for an SSN *')
 end
 
+And(/^the is primary caregiver question should display$/) do
+  expect(page).to have_css('#is_primary_caregiver_no')
```

Comment:
@jacobkagon let's use the page object model here as well as `data-cuke` attributes to target them

more info here: https://sites.google.com/ideacrew.com/ideacrew-playbook/software-development/tests